packages/node/package.json

```diff
@@ -7,9 +7,9 @@
   "license": "MIT",
   "scripts": {
     "build": "tsc -p . && rollup -c",
-    "test": "yarn run test:no-ganache && yarn run test:with-ganache",
-    "test:with-ganache": "./scripts/test.sh",
-    "test:no-ganache": "jest --testPathIgnorePatterns=test/machine/integration --setupFiles dotenv-extended/config --runInBand --bail --forceExit",
+    "test": "yarn run test:machine && yarn run test:node",
+    "test:machine": "./scripts/test.sh",
```

Comment:
These are not the right names since they just check on `test/machine/integration`

Reply:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll actually just noticed that the existing script names are totally accurate either since the node tests do in fact spin up a ganache instance.

I'll update the naming again

Reply:
Updated